### ISSUE TITLE
hash of the disclaimer text needs to be used

### ISSUE THREAD

**Issue by maks** [Western-Health-Covid19-Collaboration/wh_covid19_app#234]
Instead of setting a incremented integer as the value, we need to use a hash of the disclaimer text (eg sha1) as the value to be certain of the exact disclaimer text that a user has accepted. 

As a bonus: if this value is then reported as a analytics event along with a unique id for the device we had an auditable way to know every user is accepting specific disclaimer text.

**Comment by workerbee22** [comment]
Do we need a general issue to sort out the overlap between: Disclaimer 'top messages', Full Disclaimer and TestFlight release 'management' text. A these are all doing similar things.

**Comment by maks** [comment]
@workerbee22 yep that can be tracked in #237

**Comment by workerbee22** [comment]
While we sort out  #237 I still think the Disclaimer version flag can be converted over to a hash, independent of 237.

**Comment by workerbee22** [comment]
@maks Good idea will add as an Analytics event along with some more events and adjustments to Analytics, now that we have data coming through from people using the app.
